src/pretix/control/forms/orders.py

```diff
@@ -215,3 +215,30 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         locale_names = dict(settings.LANGUAGES)
         self.fields['locale'].choices = [(a, locale_names[a]) for a in self.instance.event.settings.locales]
+
+
+class OrderMailForm(forms.Form):
+    subject = forms.CharField(
+        label=_("Subject"),
+        required=True
+    )
+
+    def __init__(self, *args, **kwargs):
+        order = kwargs.pop('order')
+        super().__init__(*args, **kwargs)
+        self.fields['sendto'] = forms.EmailField(
+            label=_("Recipient"),
+            required=True,
+            initial=order.email
+        )
+        self.fields['sendto'].widget.attrs['readonly'] = 'readonly'
+        self.fields['message'] = forms.CharField(
+            label=_("Message"),
+            required=True,
+            widget=forms.Textarea,
+            initial=order.event.settings.mail_text_order_custom_mail,
```

Comment:
we should probably use ``order.event.settings.mail_text_order_custom_mail.localize(order.locale)`` here